src/utils/index.js

```diff
@@ -91,22 +91,20 @@ export const blockToDate = (goalBlock, height) => {
   return new Date(diff * 180000 + Date.now());
 };
 
-export const wrapTry = async (promise, store) => {
+// eslint-disable-next-line consistent-return
+export const wrapTry = async (promise) => {
   try {
     return promise.then((res) => {
-      if (!res) {
-        if (store) { store.commit('setBackendStatus', false); }
-        return null;
+      // if (!res) {
+      //   throw new Error('No response object');
+      // }
+      if (res.status !== 200) {
```

Comment:
what is the difference between `res.status !== 200` and `!res.ok`?

Reply:
There is no difference 
```
The ok read-only property of the Response interface contains a Boolean stating whether the response was successful (status in the range 200-299) or not.
```